# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mdadm/mdadm-3.2.5-do-not-include-ansidecl.h.patch.cross
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Cross-compiling mdadm fails in the case binutils was only build in stage 0.
The ansidecl.h is is only installed if the --enable-shared configure option
is given when building binutils which is not the case at toolchain stage
(aka stage 0). But even if we provide the --enable-shared configure option
at toolchain stage, the headers are installed at place which is not known
when building packages at stage 1.

--- mdadm-3.2.5/sha1.h.orig	2012-07-25 10:49:35.487768058 +0200
+++ mdadm-3.2.5/sha1.h	2012-07-25 10:50:51.385378523 +0200
@@ -26,8 +26,6 @@
 # include <limits.h>
 #endif
 
-#include "ansidecl.h"
-
 /* The following contortions are an attempt to use the C preprocessor
    to determine an unsigned integral type that is 32 bits wide.  An
    alternative approach is to use autoconf's AC_CHECK_SIZEOF macro, but
